device/thunder_ripple_sdk/src/processors/thunder_package_manager.rs

```diff
@@ -40,6 +40,9 @@ use crate::{
 use ripple_sdk::api::device::device_apps::AppMetadata;
```

Comment:
Although the ask was to add support specifically for PackageManager I did look into how difficult it would be to add observability in a more general fashion at the thunder client level. There are definitely some hurdles there, IIRC the biggest is getting MetricsContext scope at that level. When we start instrumenting more thunder APIs I think we should first spend a little more time looking into that solution.